src/Me/Navigation/Navbar.tsx

```diff
@@ -43,11 +44,12 @@ const Navbar = () => {
           src={`${process.env.NEXT_PUBLIC_PUBLIC_URL}/codingcoach-logo-192.png`}
           alt="Logo"
         />
-        <MenuItem to="/me" icon={IconHome} label="Home" />
-        <MenuItem to="/me/requests" icon={Mentorships} label="Mentorships" />
+        <MenuItem to={urls.me.get()} icon={IconHome} label="Home" />
+        <MenuItem to={urls.me.requests.get()} icon={Mentorships} label="Mentorships" />
+        {/* TODO: Hook up to new root URL */}
```

Comment:
#896 contains this new `root` route.  We can merge that one first and then patch up this PR.